lib/modules/manager/woodpecker/readme.md

```diff
@@ -0,0 +1,9 @@
+Extracts all Docker images from Woodpecker Pipeline YAML files.
+
+- [Woodpecker homepage](https://woodpecker-ci.org/)
+- [Woodpecker Docs: Pipeline Syntax](https://woodpecker-ci.org/docs/usage/pipeline-syntax)
+- [`woodpecker-ci` JSON schema](https://raw.githubusercontent.com/woodpecker-ci/woodpecker/master/pipeline/schema/schema.json)
+
+- [Section where Dependencies in YAML files are being used [Documentation]](https://woodpecker-ci.org/docs/usage/pipeline-syntax#image)
```

Comment:
We have two links that go to `docs/usage/pipeline-syntax` but one link has the `#image` fragment in the link. Maybe we can use just the first pipeline docs link and explain that the reader should also check the section about "Where dependencies in YAML files are being used"?

Our docs feel like: "Have some links, you'll figure it out.". Is this what we intend for the Woodpecker manager docs? Do we want some examples in our docs maybe?

Reply:
How about the new version?

Reply:
I like it! 😉